text/metrics/0156-counter-gauge-model.md

```diff
@@ -0,0 +1,356 @@
+# Counter, UpDownCounter, and Gauge instruments explained
+
+Counter and Gauge instruments are different in the ways they convey
+meaning, and they are interpreted in different ways.  Attributes
+applied to metric events enable further interpretation.  Because of
+their semantics, the interpretive outcome of adding an attribute for
+Counter and Gauge instruments is different.
+
+With Counter instruments, a new attribute can be introduced with
+additional measurements to subdivide a variable count.
+
+With Gauge instruments, a new attribute can be introduced with
+additional measurements to make multiple observations of a variable.
+
+The OpenTelemetry Metrics API introduces a new kind of instrument, the
+UpDownCounter, that behaves like a Counter, meaning that attributes
+subdivide the variable being counted, but their primary interpretation
+is like that of a Gauge.
```

Comment:
I find this a bit hard to understand.

_The OpenTelemetry Metrics API introduces a new kind of instrument, the ￼UpDownCounter, that behaves like a Counter,_

Perfect, so far so good :+1: 

_meaning that attributes subdivide the variable being counted,_

It starts getting a bit confusing here. Who do these attributes belong to? I assume they belong to the UpDownCounter instrument. The _behaves like a Counter, meaning that attributes ￼subdivide the variable being counted_ tells me that the Counter attributes subdivide the variable being counted, and since UpDownCounter behaves like a Counter, then I understand that UpDownCounter has attributes that also subdivide the variable being counted.

_but their primary interpretation ￼is like that of a Gauge._

Ok, here it gets confusing. the usage of "but" here makes me understand that the difference between Counter and UpDownCounter is that the primary interpretation of the latter is like that of a Gauge. What does that mean? The difference between the words Counter and UpDownCounter is "UpDown" so, is the "UpDown" what a Gauge does? :shrug: I was expecting something like this instead _but the UpDownCounter is non monotonic_.